src/Query/Builder.php

```diff
@@ -1446,16 +1447,26 @@ public function __call($method, $parameters)
         return parent::__call($method, $parameters);
     }
 
-    /** @internal This method is not supported by MongoDB. */
+    /**
+     * Return the JSON representation of the MongoDB Query.
+     * Naming is inherited and used for compatibility with third-party packages.
+     *
+     * @internal
```

Comment:
Should this still be `@internal`?

I don't have enough context to understand how/why [rappasoft/laravel-livewire-tables](https://github.com/rappasoft/laravel-livewire-tables) needs to support MongoDB, but isn't there some concern that returning JSON strings here might break _other_ packages that expect these methods to return actual SQL statements?

My thinking is that packages wishing to support this library should be the ones to change and conditionally call `toMql()`.

Reply:
It's used for debugging: https://tables.laravel-boilerplate.com/tailwind
But [Laravel Debugbar](https://github.com/barryvdh/laravel-debugbar) would provide the same information. 

And I agree this can lead to other problems if packages expect SQL and use the returned value as such.